app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,20 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
```

Comment:
Look into this message)
https://mate.academy/chats/244801?selectedMessageId=3129519